frontend/src/routes/index.tsx

```diff
@@ -14,6 +22,152 @@ function RouteComponent() {
       <PageText $variant="ingress">
         This is an application for managing the settings of FMU projects.
       </PageText>
+      <Documentation />
     </>
   );
 }
+
+function Documentation() {
+  return (
+    <div
+      style={{
+        display: "flex",
+        justifyContent: "flex-start",
+        flexWrap: "wrap",
+        padding: "20px",
+        gap: "15px",
```

Comment:
Suggest setting a larger gap, since the margin value on each card should be removed. The gap value should be specified using an EDS token - `x_large` seems appropriate.
```suggestion
        gap: tokens.spacings.comfortable.x_large,
```

Note that `tokens` needs to be imported from EDS.